src/js/post-911-gib-status/containers/Main.jsx

```diff
@@ -37,7 +38,8 @@ export class Main extends React.Component {
         break;
       case 'unavailable':
       default:
-        appContent = systemDownMessage;
+        // appContent = systemDownMessage;
+        appContent = backendErrorMessage();
```

Comment:
I'm actually not sure _when_ this should be shown?

@kreek Do you have any insight?